tests/features/test_config_tree.py

```diff
@@ -139,22 +140,18 @@ def test_deprecated_config(project: Project) -> None:
     stdout, stderr = result.communicate()
     assert result.returncode == 0, stderr
 
-    desired_logs = {
-        (
-            f"py.warnings              WARNING {project._test_project_dir}/libs/terraform/plugins/__init__.py:469: "
-            f"DeprecationWarning: The usage of config '' at {project._test_project_dir}/main.cf:3 is deprecated"
-        ),
-        (
-            # prior to https://github.com/inmanta/inmanta-core/commit/64798acc8abcc3b7b31ab657f1d04e1974209d6f
-            f"py.warnings              WARNING {project._test_project_dir}/libs/terraform/plugins/__init__.py:469: "
-            "DeprecationWarning: The usage of config '' at ./main.cf:3 is deprecated"
-        ),
-    }
+    warning_regex = re.compile(
+        r"py.warnings              WARNING (.*)/terraform/plugins/__init__.py:469: "
```

Comment:
I don't think it's a good idea to include the line number here. The test case will break every time a line is added/removed from a line above line 469.

Reply:
Fair enough, I will change it

Reply:
Done! 

Future me is thanking you I guess :stuck_out_tongue: